src/schema/v2/edition_set.ts

```diff
@@ -112,7 +112,7 @@ export const EditionSetType = new GraphQLObjectType<any, ResolverContext>({
 
         // If its for sale (and no price), return 'Available'.
```

Comment:
Minor: Should we just get rid of this comment above? It was already wrong before the changes on this PR (it wouldn't return `Available`, it would return `Contact for Price`).

Now it will return `sale_message` and it seems to make sense to just not have a comment here at all.

Reply:
I did not pay attention to that at all, thanks for bringing it up! Removed in [51acdbe](https://github.com/artsy/metaphysics/pull/5800/commits/51acdbea653ce386eb4273cd0404f8f935903ee8)